Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.18 as builder
+FROM golang:1.20 as builder
```

Comment:
nit: we should rather use images provided by Redhat (not related but when you already touching this)

Reply:
Let's do it in a separate PR